nautobot/docs/configuration/required-settings.md

```diff
@@ -285,56 +287,6 @@ The following environment variables may also be set for some of the above values
 
 For more details on configuring RQ, please see the documentation for [Django RQ installation](https://github.com/rq/django-rq#installation).
```

Comment:
At some point we should revisit this entire `RQ_QUEUES` section of the docs as well as in the core settings.py - I'm pretty sure we're no longer using the `check_releases`, `webhooks`, or `custom_fields` RQ queues for anything in core; though I think we kept them around for backward compatibility with plugins that might still be using them, we could probably at least remove the relevant sections of the documentation as we move forward. Not blocking for this PR, of course, just thinking aloud.